logfire/_internal/integrations/pydantic_ai.py

```diff
@@ -0,0 +1,32 @@
+from __future__ import annotations
+
+from typing import Literal
+
+from pydantic_ai import Agent
+from pydantic_ai.agent import InstrumentationSettings
+from pydantic_ai.models import Model
+from pydantic_ai.models.instrumented import InstrumentedModel
```

Comment:
Don't we check if it's imported and raise error to install?

Reply:
We do that for `opentelemetry.instrumentation.fastapi`, not `fastapi` itself etc. There's no instrumentation package here or extra here.